src/aten/UpSampleBilinear2d.cpp

```diff
@@ -0,0 +1,38 @@
+#include <ATen/Context.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <aten/sycl/UpSampleBilinear2dKernels.h>
+
+namespace at {
+Tensor& XPUNativeFunctions::upsample_bilinear2d_out(
+    const Tensor& self,
+    IntArrayRef output_size,
+    bool align_corners,
+    std::optional<double> scales_h,
+    std::optional<double> scales_w,
+    Tensor& output) {
+  native::xpu::upsample_bilinear2d_out_kernel(
+      output, self, output_size, align_corners, scales_h, scales_w);
+  return output;
+}
+
+Tensor& XPUNativeFunctions::upsample_bilinear2d_backward_out(
+    const Tensor& grad_output,
+    IntArrayRef output_size,
+    IntArrayRef input_size,
+    bool align_corners,
+    c10::optional<double> scales_h,
+    c10::optional<double> scales_w,
+    Tensor& grad_input) {
```

Comment:
Lack of meta process,
<img width="946" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/620b3ef8-6880-45d6-9ee5-775f912b8564">